examples/composer-shared-vpc/backend.tf

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
As it is this makes your initial apply fail. I'd rename this file backen.tf.sample or something similar

Reply:
Thanks @juliocc for review. Done